### ISSUE TITLE
how to change the energy convergence criteria "SPHInX"

### ISSUE THREAD

**Issue by mostafajoulaian** [pyiron/pyiron#931]
I need to change the energy convergence criteria for SPHInX via pyiron from the default value.
does anyone know how should I do it?

**Comment by jan-janssen** [comment]
```
job.set_convergence_precision(ionic_energy_tolerance=None, ionic_force_tolerance=None, ionic_energy=None, electronic_energy=None, ionic_forces=None)
```
https://github.com/pyiron/pyiron/blob/master/pyiron/sphinx/base.py#L942
A similar question was recently discussed in https://github.com/pyiron/pyiron/issues/925

**Comment by jan-janssen** [comment]
@mostafajoulaian Does this solve your issue? and if the issue was solved please close it.

**Comment by mostafajoulaian** [comment]
Thanks million

**Comment by mostafajoulaian** [comment]
I look at the code and I noticed we have {ionic_energy_tolerance=None, ionic_force_tolerance=None, ionic_energy=None, electronic_energy=None, ionic_forces=None} as inputs:

https://github.com/pyiron/pyiron/blob/master/pyiron/sphinx/base.py#L942

however in the calculation, I want to set number for
"electronic_energy_tolerance" but in the code, I got an error such as :
{set_convergence_precision() got an unexpected keyword argument 'ionic_energy_tolerance'}


and when I look more closely, I realized: job_sphinx.set_convergence_precision has only these three inputs (
 ionic_energy=None,
 electronic_energy=None,
 ionic_forces=None,
)
now I am a little bit confused would you please clarify it for me: 
in the attachment, you will find a piece of code and error.
<img width="1013" alt="Screenshot 2020-09-03 at 20 14 54" src="https://user-images.githubusercontent.com/62750893/92137644-9a1b4a80-ee22-11ea-8fab-414734c8fc6c.png">

**Comment by sudarsan-surendralal** [comment]
It seems that this pyiron version on the cluster isn't updated to include the newly introduced `ionic_force_tolerance` parameter. @mostafajoulaian, temporarily you can use the parameter `ionic_energy` to set the same thing (don't set the other parameters). 

@jan-janssen should we update the cluster verion already?

**Comment by jan-janssen** [comment]
@sudarsan-surendralal That is what I want to discuss in tomorrows meeting. On the cluster we run the conda version, so we have to decide if we update the conda version to 0.3.0. It would include the switch to ASE and the updates to the Sphinx class.

**Comment by jan-janssen** [comment]
I released 3.0 - announcement https://github.com/pyiron/pyiron/issues/939 - it includes all the changes of the current master branch.

**Comment by mostafajoulaian** [comment]
thanks a lot.
